requirements/base.txt

```diff
@@ -202,13 +202,13 @@ wcwidth==0.2.13
     # via prompt-toolkit
 web-fragments==2.2.0
     # via xblock
-webob==1.8.7
+webob==1.8.8
     # via xblock
-xblock==4.0.1
+xblock==5.1.0
```

Comment:
5.1.0 fixes https://github.com/openedx/XBlock/pull/778, but I still want to wait until [edx-platform removes its constraint and uses 5.1.0](https://github.com/openedx/edx-platform/blob/master/requirements/constraints.txt#L148) before upgrading this package here.